Description: Downgrade CMake version
 Downgrade required CMake version to fit wheezy-backports.
Author: Zenon Mousmoulas <zmousm@noc.grnet.gr>

Index: srt/CMakeLists.txt
===================================================================
--- srt.orig/CMakeLists.txt
+++ srt/CMakeLists.txt
@@ -7,7 +7,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 
-cmake_minimum_required (VERSION 2.8.12 FATAL_ERROR)
+cmake_minimum_required (VERSION 2.8.11 FATAL_ERROR)
 # XXX This can be potentially done in future, but there still exist
 # some dependent project using cmake 2.8 - this can't be done this way.
 #cmake_minimum_required (VERSION 3.0.2 FATAL_ERROR)
@@ -460,9 +460,9 @@ if (srt_libspec_shared)
 	set_property(TARGET ${TARGET_srt}_shared PROPERTY OUTPUT_NAME ${TARGET_srt})
     set_target_properties (${TARGET_srt}_shared PROPERTIES VERSION ${SRT_VERSION} SOVERSION ${SRT_VERSION_MAJOR})
 	list (APPEND INSTALL_TARGETS ${TARGET_srt}_shared)
-	target_link_libraries(${TARGET_srt}_shared PRIVATE ${SSL_LIBRARIES})
+	target_link_libraries(${TARGET_srt}_shared LINK_PRIVATE ${SSL_LIBRARIES})
 	if (MICROSOFT)
-		target_link_libraries(${TARGET_srt}_shared PRIVATE ws2_32.lib)
+		target_link_libraries(${TARGET_srt}_shared LINK_PRIVATE ws2_32.lib)
 	endif()
 endif()
 
@@ -484,9 +484,9 @@ if (srt_libspec_static)
 	endif()
 
 	list (APPEND INSTALL_TARGETS ${TARGET_srt}_static)
-	target_link_libraries(${TARGET_srt}_static PRIVATE ${SSL_LIBRARIES})
+	target_link_libraries(${TARGET_srt}_static LINK_PRIVATE ${SSL_LIBRARIES})
 	if (MICROSOFT)
-		target_link_libraries(${TARGET_srt}_static PRIVATE ws2_32.lib)
+		target_link_libraries(${TARGET_srt}_static LINK_PRIVATE ws2_32.lib)
 	endif()
 endif()
 
@@ -527,7 +527,7 @@ endif()
 
 foreach(tar ${srtpack_libspec_common})
 	message(STATUS "ADDING TRANSITIVE LINK DEP to:${tar} : ${PTHREAD_LIBRARY} ${dep}")
-	target_link_libraries (${tar} PUBLIC ${PTHREAD_LIBRARY} ${dep})
+	target_link_libraries (${tar} LINK_PUBLIC ${PTHREAD_LIBRARY} ${dep})
 endforeach()
 
 
@@ -542,7 +542,7 @@ endif()
 
 if (srt_libspec_shared)
 if (MICROSOFT)
-    target_link_libraries(${TARGET_srt}_shared PUBLIC Ws2_32.lib)
+    target_link_libraries(${TARGET_srt}_shared LINK_PUBLIC Ws2_32.lib)
 endif()
 endif()
 
